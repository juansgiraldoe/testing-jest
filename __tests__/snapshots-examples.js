//Snapshots.

const cliente = {
  nombre: 'Juan Sebastian.',
  balance: 500,
  tipo: 'Premium',
};

describe('Testing al cliente.', () => {
  test('Es Juan', () => {
    expect(cliente).toMatchSnapshot();
  });
});
