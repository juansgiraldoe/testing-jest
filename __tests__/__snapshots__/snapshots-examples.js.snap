// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente. Es Juan 1`] = `
{
  "balance": 500,
  "nombre": "Juan Sebastian.",
  "tipo": "Premium",
}
`;
